Create it

albanobattistella opened the pull request.

albanobattistella committed:
Create croquet_it.properties

albanobattistella committed:
Update croquet_it.properties

albanobattistella committed:
Create GalleryNames_it.properties

albanobattistella committed:
Update GalleryNames_it.properties

albanobattistella committed:
Update GalleryNames_it.properties

albanobattistella committed:
Update GalleryNames_it.properties

albanobattistella committed:
Update GalleryNames_it.properties

dabeshouse commented:
Thank you for the contribution! You have the distinction of making the first contribution to our repository from outside of the base Alice team. 

Providing these translations helped reveal deficiencies in current tool (It is slowly shutting down), and provided further impetus for us to migrate away.

Alice translations are moving to Lokalise. Once we have things straight there, the project should be under [Alice 3 Main IDE](https://app.lokalise.com/project/342638676005ef256884d4.09313519).
I have added your translations there and they will show up here in the next pull to the develop branch (more on that below).
We are still ramping up on Lokalise, but once complete we hope most translation work will be done through their web interface. We welcome any help translating and reviewing translations.

Your PR, sensibly, used our repository default and is against master. The pattern of work we try to follow here has only release code going to master, so the default target should be the develop branch. I have changed that in the repository here. I hope this won't cause any problems in your fork.

Once your translations are merged to develop from Lokalise I will close this PR.

For the translations and the motivation I must say grazie.

albanobattistella commented:
HI,
thank you for answer.
I'll waiting your merge.
Regards,
Albano
________________________________
Da: Daniel Abeshouse <notifications@github.com>
Inviato: giovedì 21 gennaio 2021 23:05
A: TheAliceProject/alice3 <alice3@noreply.github.com>
Cc: albanobattistella <albano_battistella@hotmail.com>; Author <author@noreply.github.com>
Oggetto: Re: [TheAliceProject/alice3] Create it (#164)


Thank you for the contribution! You have the distinction of making the first contribution to our repository from outside of the base Alice team.

Providing these translations helped reveal deficiencies in current tool (It is slowly shutting down), and provided further impetus for us to migrate away.

Alice translations are moving to Lokalise. Once we have things straight there, the project should be under Alice 3 Main IDE<https://app.lokalise.com/project/342638676005ef256884d4.09313519>.
I have added your translations there and they will show up here in the next pull to the develop branch (more on that below).
We are still ramping up on Lokalise, but once complete we hope most translation work will be done through their web interface. We welcome any help translating and reviewing translations.

Your PR, sensibly, used our repository default and is against master. The pattern of work we try to follow here has only release code going to master, so the default target should be the develop branch. I have changed that in the repository here. I hope this won't cause any problems in your fork.

Once your translations are merged to develop from Lokalise I will close this PR.

For the translations and the motivation I must say grazie.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/TheAliceProject/alice3/pull/164#issuecomment-764971106>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AIJS6FERGSGZYHUXVJP6BLTS3CQKJANCNFSM4V4QSWQA>.

dabeshouse commented:
Your gallery name additions are in our develop branch. Thank you for the contributions. The youtube export has been removed from Alice in favor of external tools. It had been neglected for a little too long.

There is now a wiki page describing [our new translation tool and how to contribute](https://github.com/TheAliceProject/alice3/wiki/Contributing-to-Translation-Efforts) if you happen to have more motivation in this direction.

I am closing this now that it has been folded in. Thanks for the help.
